### ISSUE TITLE
Ink compiler error with version 0.3.5

### ISSUE THREAD

**Issue by twheinen** [inkle/ink#111]
After updating to the new version 0.3.5, I am getting a compiler error when trying to compile an ink file.

```
Error	1	Unhandled error occurred compiling Ink file Ink.UnityIntegration.InkFile! Please report following error as a bug:

Unhandled Exception: System.IO.FileNotFoundException: Could not load file or assembly 'Newtonsoft.Json, Version=8.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed' or one of its dependencies. The system cannot find the file specified.
   at Ink.Runtime.Story.ToJsonString(Boolean indented)
   at Ink.CommandLineTool..ctor(String[] args)
   at Ink.CommandLineTool.Main(String[] args)
	Assets/Plugins/Ink/Editor/Compiler/InkCompiler.cs	173	1
```

The only thing I could think to try was to update Newtonsoft.Json.dll to the latest version 8.0.3, since the version packaged with ink seems to be 7.0.1. However, updating the dll didn't fix the problem, and I'm unsure what else could be causing this error. I freshly imported the unitypackage, so all the files should be where they need to be.

**Comment by twheinen** [comment]
After some trial and error, I was able to get compiling to work by placing the .NET 3.5 Newtonsoft.Json.dll version 8.0.3 in the same folder as inklecate_win.exe. Not exactly sure what caused everything to break suddenly, but I guess this works for now.

**Comment by tomkail** [comment]
Sorry for not getting back sooner - somehow this isn't happening on our machines (macs?), but we've heard plenty of reports. Until it's fixed properly, that's our recommended solution, which we've pushed.

**Comment by joethephish** [comment]
I believe this all stems from moving over to trying to use nuget for packages (which includes Newtonsoft.Json.dll), and on Xamarin it's not as well supported on Mac. It may be failing to get the right version, or the build script isn't working correctly. We'll try investigating more fully soon...

Side note: I'd dearly love to remove our JSON dependency, it's causing us no end of issues!

**Comment by joethephish** [comment]
Now fixed! See [version 0.4](https://github.com/inkle/ink/releases/tag/0.4.0) - Newtonsoft.Json.Net dependency removed (I hand-rolled my own JSON serialiser - turned out to be rather easy and fast to implement!)
